tests/bats/README.md

```diff
@@ -0,0 +1,71 @@
+**Warning:** the test suite runs _invasively_ against the Kubernetes cluster that your local `kubectl` is currently configured against.
+
+## Usage
+
+Invoke `make bats` in the root of this repository.
+
+
+### Test local dev state (artifacts not pushed)
+
+Not yet supported.
+Let's change this ASAP.
+
+This test suite for now assumes public availability of a Helm chart on GHCR or NGC, pointing to a container image publicly available on GHCR or NGC.
+
+### Test Helm chart from registery
+
+#### Default versions
+
+Say, this is the current local git revision:
+
+```console
+$ git rev-parse --short=8 HEAD
+e6e1dde4
+```
+
+Then the test suite runs with the default configuration, for example:
+```console
+$ make bats
+...
+        --env TEST_CHART_REPO="oci://ghcr.io/nvidia/k8s-dra-driver-gpu" \
+        --env TEST_CHART_VERSION=25.8.0-dev-e6e1dde4-chart \
+        --env TEST_CHART_LASTSTABLE_REPO="oci://ghcr.io/nvidia/k8s-dra-driver-gpu" \
+        --env TEST_CHART_LASTSTABLE_VERSION="25.3.2-7020737a-chart" \
+        --env TEST_CRD_UPGRADE_TARGET_GIT_REF=e6e1dde4 \
+...
+12 tests, 0 failures in 166 seconds
+```
```

Comment:
What exactly is TEST_CHART_LASTSTABLE_VERSION and how is it calculated? Does it map to the SHA of one of our release tags (i.e. the "latest" one)?

What happens if we release `v25.8.0` and then we go to release / test `v25.3.3`. Does it test against `v25.3.2` or `v25.8.0` as `TEST_CHART_LASTSTABLE_VERSION`?

Reply:
> What exactly is TEST_CHART_LASTSTABLE_VERSION and how is it calculated?

Today, this is what we want to test today.

Next, I'd like for this to be determined automatically but I don't know exactly how yet. Also see https://github.com/NVIDIA/k8s-dra-driver-gpu/pull/572#discussion_r2358633091